src/files/FileEntity.js

```diff
@@ -1,29 +1,35 @@
-import { Entity } from '../es6/Entity';
-
-/**
- * FileEntity class representation of a d2l content file entity.
- */
-export class FileEntity extends Entity {
-
-	/**
-	 * @returns {string|null} File's location
-	 */
-	getFileLocationHref() {
-		if (!this._entity || !this._entity.hasLinkByClass('filtered-data')) {
-			return null;
-		}
-
-		return this._entity.getLinkByClass('filtered-data').href;
-	}
-
-	/**
-	 * @returns {string|null} File's raw data location
-	 */
-	getFileDataLocationHref() {
-		if (!this._entity || !this._entity.hasLinkByClass('raw-data')) {
-			return null;
-		}
-
-		return this._entity.getLinkByClass('raw-data').href;
-	}
-}
+import { Entity } from '../es6/Entity';
+
+/**
+ * FileEntity class representation of a d2l content file entity.
+ */
+export class FileEntity extends Entity {
+	/**
+	 * @returns {string|undefined} Title of the content HTML file template entity
+	 */
+	title() {
```

Comment:
This is mostly whitespace, but this is the real change. I had to move the `title` to here because the sub-entities on the template entity are `FileEntity` objects, not `TemplateEntity` objects